test/puppetlabs/puppetdb/cli/import_export_roundtrip_test.clj

```diff
@@ -202,3 +202,15 @@
   (check-invalid-url-handling
    #(#'import/main "--host" "local:host" "--infile" "/dev/null" "--port" 10000)
    #"^Invalid destination .*"))
+
+(deftest changing-queue-destination-name
```

Comment:
Not sure where the best place is to put this test and the disable jmx test below. I put this one here as it needs stuff to submit a command etc. The jmx one did not, so I put it alongside the other mq related tests.